sdk/keyvault/Azure.Security.KeyVault.Keys/README.md

```diff
@@ -52,21 +52,21 @@ Use the [Azure CLI][azure_cli] snippet below to create/get client secret credent
         "tenant": "tenant-ID"
     }
     ```
-* Use the returned credentials above to set  **AZURE_CLIENT_ID**(appId), **AZURE_CLIENT_SECRET**(password) and **AZURE_TENANT_ID**(tenant) environment variables. The following example shows a way to do this in Powershell:
+* Use the returned credentials above to set  **AZURE_CLIENT_ID** (appId), **AZURE_CLIENT_SECRET** (password) and **AZURE_TENANT_ID** (tenantId) environment variables. The following example shows a way to do this in Powershell:
```

Comment:
If the parentheticals are to clarify the environment variables, then they should be separate words like "application ID" and "tenant ID". Using variable-cased names doesn't really clarify them.

Reply:
Using 'appId', 'tenant' to keep consistency with the above [output ](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/keyvault/Azure.Security.KeyVault.Keys#createget-credentials)content would be better. So that a new learner can know what are these environment variables smoothly. what do you think @heaths ?